core/src/Revolution/Processors/Element/Get.php

```diff
@@ -22,6 +22,61 @@
  */
 abstract class Get extends GetProcessor
 {
+    protected $hasStaticFile = false;
+
+    public function initialize()
+    {
+        // Intitializing parent first, as we need the Element object created before moving forward
+        if (parent::initialize()) {
+            $className = array_pop(explode('\\', $this->classKey));
```

Comment:
Only variables can be passed by reference. You need to introduce a variable here and pass that to `array_pop`.